### DISCUSSION TITLE
Free Session and the Current Interval Accumulative Consumption Electricity sensor

### DISCUSSION THREAD

**Discussion by deltamelter** [BottlecapDave/HomeAssistant-OctopusEnergy#1035] [General]
I think this is new fairly, right? I was just looking for the consumption for the hour so far and was wondering why the chart is so gappy?
![image](https://github.com/user-attachments/assets/c4aea8e1-6e83-4824-870d-91ee109fe25e)

**Comment by BottlecapDave** [comment]
The current interval accumulative consumption sensor shows the current period's available consumption. If the consumption data isn't available for the current period, it currently returns unavailable because the data is not available instead of reporting zero (which is arguably wrong). Depending on how oftern your Home Mini updates, this could be producing the gappy data you're seeing. Free electricity session data isn't currently available yet in the integration (but I'm working on it if you look hard enough ;))
